hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/AbstractHoodieWriteClient.java

```diff
@@ -1037,6 +1038,10 @@ protected void setWriteSchemaForDeletes(HoodieTableMetaClient metaClient) {
     }
   }
 
+  protected void unpersistRDD() {
```

Comment:
Can we just do this in `close()` instead of introducing a new api here?

regardless, we need to call this something spark agnostic since its in client-common.

Reply:
`Upersist` has to been executed immediately before running inline compaction or clustering to release the memory and disk. Therefore, cannot move it into `close`

Reply:
+1 for renaming method to something spark agnostic.
@minihippo This is nearing landing. Can you please address the comment?

Reply:
Made this fix myself. will land after CI passes